test/unit/lib/tools.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { afterAll, beforeAll, describe, expect, test } from 'vitest'
+import { shuffle } from '../../../src/lib/tools'
+
+describe('shuffle', () => {
+    const testArray = [1,2,3,4,5,6]
```

Comment:
Ah, I guess we're missing `tsx?` files in the `lint-staged` config.

Reply:
looks like it, although I am not too familiar :)